upgrading.md

```diff
@@ -2,6 +2,77 @@
 
 This documentation is meant to help you upgrade across versions, when potentially breaking changes are introduced.
 
+## v1.4.6
```

Comment:
this will generate the v1.4.8 right?

Reply:
you're so right! 😄

Reply:
Corrected in 7140b6ec3557a1878ae4541521e15cb80daa95ae :)